mappings/empire_v2.0.yaml

```
@@ -0,0 +1,341 @@
+- Hungary>Serbia  
+- Hungary>Slovakia  
+- Hungary>Slovenia  
+- Italy>Slovenia  
+- Latvia>Sweden  
+- Lithuania>Latvia  
+- Lithuania>Poland  
+- Lithuania>Sweden  
+- Macedonia>Serbia  
+- Moray Firth>Borssele  
+- Moray Firth>Dogger Bank  
+- Moray Firth>East Anglia  
+- Moray Firth>Firth of Forth  
+- Moray Firth>Helgolander Bucht  
+- Moray Firth>Hollandsee Kust  
+- Moray Firth>Hornsea  
+- Moray Firth>Nordsoen  
+- Moray Firth>Norfolk  
+- Moray Firth>Outer Dowsing  
+- Moray Firth>Sørliğe Nordsjø I  
+- Moray Firth>Sørliğe Nordsjø II  
+- Moray Firth>Utsira Nord  
+- Netherlands>Borssele  
+- Netherlands>Helgolander Bucht  
+- Netherlands>Hollandsee Kust  
+- Netherlands>NO2  
+- NO1>NO2  
+- NO1>NO3  
+- NO1>NO5  
+- NO1>Sweden  
+- NO2>NO5  
+- NO2>Sørliğe Nordsjø I  
+- NO2>Sørliğe Nordsjø II  
+- NO2>Utsira Nord  
+- NO3>NO4  
+- NO3>NO5  
+- NO3>Sweden  
+- NO4>Sweden  
+- Nordsoen>Sørliğe Nordsjø I  
+- Nordsoen>Sørliğe Nordsjø II  
+- Nordsoen>Utsira Nord  
+- Norfolk>Borssele  
+- Norfolk>East Anglia  
+- Norfolk>Helgolander Bucht  
+- Norfolk>Hollandsee Kust  
+- Norfolk>Nordsoen  
+- Norfolk>Sørliğe Nordsjø I  
+- Norfolk>Sørliğe Nordsjø II  
+- Norfolk>Utsira Nord  
+- Outer Dowsing>Borssele  
+- Outer Dowsing>East Anglia  
+- Outer Dowsing>Helgolander Bucht  
+- Outer Dowsing>Hollandsee Kust  
+- Outer Dowsing>Nordsoen  
+- Outer Dowsing>Norfolk  
+- Outer Dowsing>Sørliğe Nordsjø I  
+- Outer Dowsing>Sørliğe Nordsjø II  
+- Outer Dowsing>Utsira Nord  
+- Poland>Slovakia  
+- Poland>Sweden  
+- Romania>Serbia  
+- Sørliğe Nordsjø I>Sørliğe Nordsjø II  
+- Spain>France  
+- Spain>Ireland  
+- Spain>Portugal  
+- Switzerland>France  
+- Switzerland>Germany  
+- Switzerland>Italy  
+- Utsira Nord>Sørliğe Nordsjø I  
+- Utsira Nord>Sørliğe Nordsjø II  
+
+
+common_regions:
+  # includes all native regions
+  - Europe:
+      - Austria
+      - Bosnia and Herzegovina
+      - Belgium
+      - Bulgaria
+      - Switzerland
+      - Czech Republic
+      - Germany
+      - Denmark
+      - Estonia
+      - Spain
+      - Finland
+      - France
+      - United Kingdom
+      - Greece
+      - Croatia
+      - Hungary
+      - Ireland
+      - Italy
+      - Lithuania
+      - Luxembourg
+      - Latvia
+      - North Macedonia
+      - The Netherlands
+      - Norway
+      - Poland
+      - Portugal
+      - Romania
+      - Serbia
+      - Sweden
+      - Slovenia
+      - Slovakia
+  - EU27 & UK:
+      - Austria
+      - Belgium
+      - Bulgaria
+      - Croatia
+#      - Cyprus - missing in the model
+      - CzechR
+      - Denmark
+      - Estonia
+      - Finland
+      - France
+      - Germany
+      - Greece
+      - Hungary
+      - Ireland
+      - Italy
+      - Latvia
+      - Lithuania
+      - Luxemb.
+#       - Malta - missing in the model
+      - Netherlands
+      - Poland
+      - Portugal
+      - Romania
+      - Slovakia
+      - Slovenia
+      - Spain
+      - Sweden
+      - GreatBrit.
+  - EU27:
```

If Malta and Cyprus are not included in the model, I suggest to use the more descriptive version (see also `european-regions.yaml`).
```suggestion
  - EU27 (excl. Malta & Cyprus):
```